dbuf-lsp/src/core/default_ast/mod.rs

```diff
@@ -0,0 +1,22 @@
+//! Provides function, that returns Ast sample.
```

Comment:
I think we can add a `FIXME` or a `TODO` comment to remove the module once we have a parser integrated